airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/LabelInfo.tsx

```diff
@@ -26,6 +27,33 @@ const Description: React.FC<Pick<LabelInfoProps, "label" | "description">> = ({
   );
 };
 
+const Options: React.FC<Pick<LabelInfoProps, "options">> = ({ options }) => {
+  if (!options) {
+    return null;
+  }
+
+  return (
+    <div>
+      <h5 className={styles.optionsHeader}>
```

Comment:
I agree that the `Options` title feels too small - maybe we should use a larger heading for the tooltip tile (e.g. h3) and use h4 here instead